avro-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaAvroSerDeConfig.java

```diff
@@ -23,7 +23,8 @@
 import io.confluent.kafka.serializers.subject.TopicNameStrategy;
 
 /**
- * @deprecated  Use {@link io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig} instead
+ * TODO: deprecate this class
+ * Use {@link io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig} instead
```

Comment:
Why did you remove `@deprecated`?

Reply:
KSQL is compiling with `-Werror` and so deprecation warnings cause the build to fail.  Once this PR is merged, I will move KSQL to the new APIs, and then I can add back the `@deprecated` tag.